examples/server/server.cpp

```diff
@@ -2173,7 +2183,24 @@ static void server_params_parse(int argc, char ** argv, server_params & sparams,
                }
             }
             key_file.close();
-        } else if (arg == "--timeout" || arg == "-to") {
+
+        }
```

Comment:
I kind of hate that this breaks the brace formatting of the rest of the `if/else if` chain, but I figured it was better than duplicating the `else if (arg == "--timeout"...`.